share/picongpu/tests/XrayScattering/lib/python/picongpu/checks.py

```diff
@@ -0,0 +1,81 @@
+from os.path import join
+import numpy as np
+import openpmd_api as api
+from picongpu.plugins.data import XrayScatteringData
+from is_close import is_close
+
+
+def compare_with_fft(species, bound_electrons, rotation=None):
+    simulation_path = '../../../../'
+
+    # Load pluginOutput
+    xray_scattering_data = XrayScatteringData(simulation_path, species, 'h5')
+    amplitude = xray_scattering_data.get(iteration=0)
+    del xray_scattering_data
+
+    # Load density
+    internal_path = 'simOutput/h5'
+    file_name = 'simData_%T.h5'
+    path_output = join(simulation_path, internal_path, file_name)
+    series_output = api.Series(path_output, api.Access_Type.read_only)
+    i = series_output.iterations[0]
+    e_mesh = i.meshes['e_density']
+    ed = e_mesh['\x0bScalar']
```

Comment:
Why do you need the `\x0b` in front?

Reply:
So the documentation suggests using `api.Mesh_Record_Component.SCALAR` instead of the string. I will change to that. I really don't know where this comes from but `list(i.meshes)` will  give `["\x0bScalar"]` .